pkg/keyctl/key.go

```diff
@@ -20,8 +19,8 @@ type Key struct {
 }
 
 // ID returns the 32-bit kernel identifier for a specific key
-func (k *Key) ID() int32 {
-	return int32(k.id)
+func (k *Key) ID() int {
+	return int(k.id)
```

Comment:
Strictly speaking, this is an API break. Do we need to change this? `int` would certainly be more idiomatic Go; still, the kernel `key_serial_t` is an `int32_t`, so this does not seem to hurt anything.

Same for `ID.ID` / `Keyring.ID`.

Reply:
Ah, you're right.  The APIs in `golang.org/x/sys/unix` use `int` for key types, so I changed the package to match without considering that.